kong/plugins/aws-lambda/aws-serializer.lua

```diff
@@ -54,7 +54,10 @@ return function(ctx, config)
 
   -- prepare body
   local body, isBase64Encoded
-  local skip_large_bodies = config and config.skip_large_bodies or true
+  local skip_large_bodies = true
+  if config ~= nil and config.skip_large_bodies ~= nil then
```

Comment:
```suggestion
  if config and config.skip_large_bodies ~= nil then
```

Reply:
I committed your suggestion, but I don't see how it would alter behavior to make the tests fail. I think you meant to take off the `~= nil` against the `config.skip_large_bodies`, not against the `config` part of that clause. `config` is a non-boolean value, so `config ~= nil` is functionally equivalent to `config` in that clause, whereas `config.skip_large_bodies` is a boolean value, so if you had removed the `~= nil` against that, it would have altered the meaning of the line. Is this what you meant to do @Tieske ?